### ISSUE TITLE
The accuracy of retrosynthesis reaction center prediction

### ISSUE THREAD

**Issue by yyc776** [DeepGraphLearning/torchdrug#103]
Hello,
Thanks for sharing this library! The reproduced center-identification results of[ G2Gs](https://arxiv.org/pdf/2003.12725.pdf) are different from the paper.

```
reaction class known 90.2
reaction class unknown 75.8
```

These are the reported results from the [Torchdrug tutorial](https://torchdrug.ai/docs/tutorials/retrosynthesis.html#center-identification):

The evaluation result on the validation set may look like:
accuracy: 0.836367

In my understanding, this value is calculated when the reaction class is unknown. I cannot understand why it is higher than reported in the literature. 

Thank you very much

**Comment by shichence** [comment]
Hi! Inspired by [this paper](https://arxiv.org/pdf/2006.07038.pdf), we slightly adapt the implementation of the reaction center module. In a nutshell, in the original paper, we treat center identification as a binary classification problem defined on each node in the graph, while in this repo, we treat it as a multi-class classification problem and define a cross-entropy loss on all nodes in the graph. We find the latter one works quite well.

**Comment by yyc776** [comment]
Thank you very much for your reply, which is very helpful! Wish you peace and health on Dragon Boat Festival！
